schematron/v2.2.0/tests/test_v2-2-0_L200_Audit.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+
+from tools.constants import BSYNC_NSMAP
+from tools.validate_sch import validate_schematron
+
+from schematron.conftest import AssertFailureRolesMixin, v2_2_0_SCH_DIR, exemplary_tree
+
+
+class TestL200Audit(AssertFailureRolesMixin):
+    schematron = os.path.join(v2_2_0_SCH_DIR, 'v2-2-0_L200_Audit.sch')
+    exemplary_file = os.path.join(v2_2_0_SCH_DIR, 'exemplary_files', 'L200_Audit.xml')
+
+    def test_exemplary_file_is_valid(self):
+        # -- Act
+        failures = validate_schematron(self.schematron, self.exemplary_file)
+
+        # -- Assert
+        self.assert_failure_messages(failures, {})
+
+    def test_is_invalid_when_schedule_types_differ_inside_schedule_details(self):
```

Comment:
I like this test for now.  Question becomes: if we are going to require this, should the `auc:ScheduleCategory` element actually be a child of the `auc:Schedule` ?

Reply:
I'm not sure, I guess it could make sense that you have multiple systems that share the same period (ie start and end date), which is how it's currently modeled. I ended up making this requirement because it made it easier for me to make later assertions about different schedule types